dbsnap_verify/rds_funcs.py

```diff
@@ -0,0 +1,234 @@
+from operator import itemgetter
+
+from random import choice
+
+from string import (
+    letters,
+    digits,
+)
+
+VALID_SNAPSHOT_TYPES = ["automated", "manual"]
+
+
+def generate_password(size=9, pool=None):
+    """Return a system generated password.
+    Args:
+        size (int): The desired length of the password to generate (Default 9).
+        pool (list): list of chars to choose from.
+            (Default digits and letters [upper/lower])
+    Returns:
+        str: the raw password
+    """
+    if pool == None:
```

Comment:
This could be the default value - strings are immutable, so it's safe.

Reply:
I actually thought `sting.letters` and `string.digits` where `list`, turns out they are just `str`, will fix.